features/fields-generator/fields_mapping_generator.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/env python
+
+"""
+Genereert csv bestand met alle mogelijke unieke paden naar velden in de response.
+Het bestand heeft twee kolommen: "fields pad" en "volledig pad".
+De gegenereerde lijst bevat in kolom "fields pad" alle mogelijke waarden die in de "fields" parameter ingevuld kunnen worden.
+In kolom "volledig pad" staat dan de mapping naar het veld dat moet worden geleverd.
+De kolom "fields pad" bevat het al dan niet ingekorte pad dat uniek verwijst naar één veld.
+De kolom "volledig pad" bevat het volledige pad naar het veld.
+Er kunnen meerdere rijen zijn, dus meerdere verschillende "fields pad" waarden, die verwijzen naar hetzelfde veld, dus met dezelfde waarde voor "volledig pad".
+Er zijn rijen opgenomen voor simple type velden (string, integer, boolean), en ook voor objecten (groepen velden).
+
+Deze lijst bevat ook velden die niet expliciet met fields gevraagd hoeven te worden, omdat die al automatisch geleverd zullen worden (bijv. geheimhouding, inOnderzoek en type).
+
+De csv lijst wordt opgeslagen onder de naam "fields-mapping-{schemaComponent}.csv", waarbij {schemaComponent} de naam is van de schemacomponent voor de resource in de response.
+
+Command line arguments:
+--config        bestand van projectspecifieke configuratie json bestand
+--debug   -d    toon debug info
+--info    -i    toon voortgang info
+--help    -h    toon help op command line arguments
+
+Gebruik dit door bijvoorbeeld: python fields_mapping_generator.py --config"settings.json"
+"""
+
+
+import yaml, json
```

Comment:
yaml moest worden gedownload. Misschien handig om dat in de readme aan te geven. Zeker omdat de package pyyaml heet